test/test_etl.py

```diff
@@ -576,22 +576,22 @@ def test_row_data(self):
         self.assertEqual(tbl.row_data(1), row)
 
     def test_stack(self):
-        tbl1 = self.tbl
+        tbl1 = self.tbl.select_rows(lambda x: x)
         tbl2 = Table([{'first': 'Mary', 'last': 'Nichols'}])
         # Different column names shouldn't matter for stack()
         tbl3 = Table([{'f': 'Lucy', 'l': 'Peterson'}])
         tbl1.stack(tbl2, tbl3)
 
-        expected_tbl = Table(petl.stack(tbl1.table, tbl2.table, tbl3.table))
+        expected_tbl = Table(petl.stack(self.tbl.table, tbl2.table, tbl3.table))
```

Comment:
I'm interested in understanding this change. You can't reference this through tbl1?

Reply:
`tbl1` gets modified above with the `stack` method.. but I want to access the original contents from `self.tbl`.